src/PointwiseFunctions/GeneralRelativity/Christoffel.hpp

```diff
@@ -38,4 +45,85 @@ template <size_t SpatialDim, typename Frame, IndexType Index, typename DataType>
 tnsr::abb<DataType, SpatialDim, Frame, Index> christoffel_first_kind(
     const tnsr::abb<DataType, SpatialDim, Frame, Index>& d_metric) noexcept;
 // @}
-} // namespace gr
+namespace Tags {
+/// Compute item for spacetime Christoffel symbols of the first kind
+/// \f$\Gamma_{abc}\f$ computed from the first derivative of the
+/// spacetime metric.
+///
+/// Can be retrieved using `gr::Tags::SpacetimeChristoffelFirstKind`
+template <size_t SpatialDim, typename Frame, typename DataType>
+struct SpacetimeChristoffelFirstKindCompute
+    : SpacetimeChristoffelFirstKind<SpatialDim, Frame, DataType>,
+      db::ComputeTag {
+  using argument_tags =
+      tmpl::list<DerivativesOfSpacetimeMetric<SpatialDim, Frame, DataType>>;
+  static constexpr tnsr::abb<DataType, SpatialDim, Frame,
+                             IndexType::Spacetime> (*function)(
+      const tnsr::abb<DataType, SpatialDim, Frame, IndexType::Spacetime>&) =
+      &christoffel_first_kind<SpatialDim, Frame, IndexType::Spacetime,
+                              DataType>;
+  using base = SpacetimeChristoffelFirstKind<SpatialDim, Frame, DataType>;
+  using type = typename base::type;
+};
+
+/// Compute item for spacetime Christoffel symbols of the second kind
+/// \f$\Gamma^a_{bc}\f$ computed from the Christoffel symbols of the
+/// first kind and the inverse spacetime metric.
+///
+/// Can be retrieved using `gr::Tags::SpacetimeChristoffelSecondKind`
+template <size_t SpatialDim, typename Frame, typename DataType>
+struct SpacetimeChristoffelSecondKindCompute
+    : SpacetimeChristoffelSecondKind<SpatialDim, Frame, DataType>,
+      db::ComputeTag {
+  using argument_tags =
+      tmpl::list<SpacetimeChristoffelFirstKind<SpatialDim, Frame, DataType>,
+                 InverseSpacetimeMetric<SpatialDim, Frame, DataType>>;
+  static constexpr Tensor<
+      DataType, Symmetry<2, 1, 1>,
+      index_list<SpacetimeIndex<SpatialDim, UpLo::Up, Frame>,
```

Comment:
The `SpacetimeIndex` structs are used a bunch in these function pointer declarations. This could be made more brief with a couple of type aliases for them, like `using up_index = SpacetimeIndex<SpatialDim, UpLo::Up, Frame>`. I won't insist on this change; I'll leave it up to you whether you feel it improves the struct.

Reply:
Isn't this whole thing just `tnsr::Abb<DataType, SpatialDim>`?

Reply:
replaced these long expressions with abbreviated `tnsr` aliases now.

Reply:
oh, yes, the abbreviated name is a significant improvement. :+1: